django_netjsonconfig/admin.py

```diff
@@ -15,7 +15,13 @@ class Meta(BaseForm.Meta):
         model = Template
 
 
+def clone_selected_templates(modeladmin, request, queryset):
+    for tmpl in queryset:
+        tmpl.clone(request.user)
+
+
 class TemplateAdmin(AbstractTemplateAdmin):
+    actions = [clone_selected_templates]
```

Comment:
I think you should move this code to https://github.com/openwisp/django-netjsonconfig/blob/master/django_netjsonconfig/base/admin.py

Reply:
I think I should not, this action requires quite different implementation in openwisp-controller for example when it has to display a organization-selection intermediate page.

Reply:
indeed it requires a different implementation, but openwisp-controller will override it, is better if we keep all the core logic in `base.py` since it's almost all there.